mw-webapp/src/dataAccessLogic/DTOToBusinessConverter/mentorCommentDTOToMentorCommentConverter.ts

```diff
@@ -0,0 +1,32 @@
+import {MentorComment} from "src/model/businessModel/MentorComment";
+import {User} from "src/model/businessModel/User";
+import {MentorCommentDTO} from "src/model/firebaseCollection/MentorCommentDTO";
+import {UserDTO} from "src/model/firebaseCollection/UserDTO";
+
+export const MentorCommentDTOToMentorCommentConverter = (mentorComment: MentorCommentDTO, usersDTO: UserDTO[]) => {
```

Comment:
check example and refacor it please

Reply:
I didnt understand which example and where